taskfiles/utils.yml

```
@@ -103,3 +103,47 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # C++ LINTING UTILS
+  # ===
+
+  # @param {string} FLAGS - The flags to pass to clang-format
+  # @param {string} LINT_VENV_DIR - Directory with venv fo linting
+  # @param {[]string} SRC_DIRS - The directories to start linting on.
+  # clang-format.
+  clang-format:
+    internal: true
+    ignore_error: true
+    requires:
+      vars: ["FLAGS", "LINT_VENV_DIR", "SRC_DIRS"]
+    cmds:
+      - for:
+          var: SRC_DIRS
+        cmd: |-
+          . "{{.LINT_VENV_DIR}}/bin/activate"
+          find {{.ITEM}} \
```

```suggestion
          find "{{.ITEM}}" \
```

We should quote paths.